pypeit/spectrographs/keck_mosfire.py

```diff
@@ -72,10 +72,8 @@ def default_pypeit_par(self):
         par['calibrations']['wavelengths']['fwhm']= 5.0
         par['calibrations']['wavelengths']['n_final']= 4
         par['calibrations']['wavelengths']['lamps'] = ['OH_NIRES']
```

Comment:
We should make a linelist for MOSFIRE resolution which is higher than NIRES. That will improve the fits and residuals.